Corona-Warn-App/src/main/res/layout/contact_diary_activity.xml

```diff
@@ -11,7 +11,6 @@
         android:name="androidx.navigation.fragment.NavHostFragment"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        app:defaultNavHost="true"
-        app:navGraph="@navigation/contact_diary_nav_graph" />
```

Comment:
We don't need this?

Reply:
Will be set programmatically in the ContactActivity. This allows to dynamically set the first fragment based on the data from the Shared Preferences

Reply:
Looks like it works without it but the main activity xml has it so I'm not sure...